cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
+    Please enter your date in YYYY-MM-DD format.
+reviews: []
+specification: 
+  answer: false
+  answer_note: |
+   I didn't find any relevant specifications.
+  instructions: |
+      Is there mention of a violation of a specification? For example,
+      an RFC specification, a protocol specification, or a requirements
+      specification.
+      
+      Be sure to check all artifacts for this: bug report, security
+      advisory, commit message, etc.
+      
+      The answer field should be boolean. In answer_note, please explain
+      why you come to that conclusion.
+subsystem: 
+  answer: "found in the directory of the fix commit: \"django/contrib/auth/\". a few files were modified to address password reset poisoning. b45c377f8f488955e0c7069cad3f3dd21910b071\n"
+  name: authentication
   question: |
-    Are there any common lessons we have learned from class that apply to this
-    vulnerability? In other words, could this vulnerability serve as an example
-    of one of those lessons?
-
-    Leave "applies" blank or put false if you did not see that lesson (you do
-    not need to put a reason). Put "true" if you feel the lesson applies and put
-    a quick explanation of how it applies.
-
-    Don't feel the need to claim that ALL of these apply, but it's pretty likely
-    that one or two of them apply.
-
-    If you think of another lesson we covered in class that applies here, feel
-    free to give it a small name and add one in the same format as these.
-  defense_in_depth:
-    applies:
-    note:
-  least_privilege:
-    applies:
-    note:
-  frameworks_are_optional:
-    applies:
-    note:
-  native_wrappers:
-    applies:
-    note:
-  distrust_input:
-    applies:
-    note:
-  security_by_obscurity:
-    applies:
-    note:
-  serial_killer:
-    applies:
-    note:
-  environment_variables:
-    applies:
-    note:
-  secure_by_default:
-    applies:
-    note:
-  yagni:
-    applies:
-    note:
-  complex_inputs:
-    applies:
-    note:
-mistakes:
+      What subsystems was the mistake in?
+      
+      Most systems don't have a formal list of their subsystems, but you can
+      usually infer them from path names, bug report tags, or other key words
+      used. A single source file is not what we mean by a subsystem. In Django,
+      the "Component" field on the bug report is useful. But there may be other
+      subsystems involved.
+      
+      Your subsystem name(s) should not have any dots or slashes in them. Only
+      alphanumerics, whitespace, _, - and @.Feel free to add multiple using a YAML
+      array.
+      
+      In the answer field, explain where you saw these words.
+      In the name field, a subsystem name (or an array of names)
+      
+      e.g. clipboard, model, view, controller, mod_dav, ui, authentication
+unit_tested: 
+  code: false
+  code_answer: "The surrounding subsystem does not have any related test cases regarding this bug.\n"
```

Comment:
Were the subsystem or class(es) related unit tested at all before?  If so, I believe the answer should be `true`.